packages/flame_networks_assets/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
TODO?

Reply:
I believe melos will replace all this when we do the first release, @spydon can you confirm this? And that is not the case, what should go in here?

Reply:
Melos only prepends to the file, so you can just leave it empty